# Point for centroids
float32 x
float32 y